core/lib/spree/rails_compatibility.rb

```diff
@@ -9,95 +9,6 @@ module Spree
   # versions. It's also meant to contain compatibility for features that we use
   # internally in the Solidus code base.
   module RailsCompatibility
```

Comment:
Although it could be a good idea to add it again in the future, should we remove this now useless module?

Reply:
on it, I was also unsure 👍

Reply:
Done ✂️